### ISSUE TITLE
Can't start recording events

### ISSUE THREAD

**Issue by mchr3k** [redcanaryco/mac-monitor#23]
**Describe the bug**
I've tried installing `v1.0.3` through homebrew and also using the released `.pkg` file. In both cases the behavior is the same. I am unable to start recording events even after granting permissions.

**Expected behavior**
I expect to be able to start recording events.

**To Reproduce**
1. Launch "Red Canary Mac Monitor"
2. Select Menu "Security Extension -> System Setting: Full Disk Access", enable this for "Red Canary Security Extension", confirm with TouchID, close System Settings window
3. Click greyed out "Start" button in "Red Canary Mac Monitor" window
4. Error message: "Quit and re-open! You need to quit and re-open the app after enabling Full Disk Access."
5. Click "Let's do it!" which just appears to quick the application
6. Launch "Red Canary Mac Monitor"
7. Select Menu "Security Extension -> System Setting: Full Disk Access", confirm that this is already enabled for "Red Canary Security Extension"
8. Click greyed out "Start" button in "Red Canary Mac Monitor" window
9. Get the same error message: "Quit and re-open! You need to quit and re-open the app after enabling Full Disk Access."

**Platform specifics (please complete the following information):**
 - macOS version: 13.3.1 (a) 
 - Architecture: Apple Silicon
 - Version: Version 1.0.3 (1), Build name: GoldCardinal

**Comment by Brandon7CC** [comment]
Hi @mchr3k can you please ensure that you "Allow" the System Extension in "System Settings" > "Privacy & Security"? You will see something like the screenshot below. This is also mentioned in the README. Let me know if you have any more problems getting up and running! 

<img width="827" alt="Screenshot 2023-05-03 at 7 50 59 PM" src="https://user-images.githubusercontent.com/22716543/236101335-29ba3325-dd8d-4660-86e2-73179f8bbb35.png">

**Comment by AndrewMohawk** [comment]
After going through those steps to give it access I have to do Security Extension -> Install again before the button was enabled for me.

**Comment by Brandon7CC** [comment]
When something like this happens it's important to note that the app will prompt to install the Security Extension on each app launch 😉 

Additionally, if there's something unknown going on here's some **helpful info**:

1. Ensure that you "Allow" the Security Extension when its installed (shown above). You can check all is good with the next step:

### Important
2. Verify that `/usr/bin/systemextensionsctl --list` reports that the: "Red Canary Security Extension" is both `activated` and `enabled`. When "Allow" (shown in the above) is **not clicked the Security Extension will only be shown as "activated waiting for user"**:
```
❯ systemextensionsctl list                                                                    09:18:28 PM
1 extension(s)
--- com.apple.system_extension.endpoint_security
enabled	active	teamID	bundleID (version)	name	[state]
*	*	UA6JCQGF3F	com.redcanary.agent.securityextension (1.0.3/1)	Red Canary Security Extension	[activated enabled]
```
4. Ensure that Full Disk Access is granted to the Red Canary Security Extension

If all else fails shoot me a note!

**Comment by mchr3k** [comment]
Sorry, I had misread these two steps and didn't quite understand what I needed to do for the first one:

* You'll be prompted to "Open System Settings" to "Allow" the System Extension.
* Next, System Settings will automatically open to Full Disk Access -- you'll need to flip the switch to enable this for the Red Canary Security Extension. Full Disk Access is a [requirement of Endpoint Security](https://developer.apple.com/documentation/endpointsecurity/3259700-es_new_client#:~:text=The%20user%20does%20this%20in%20the%20Security%20and%20Privacy%20pane%20of%20System%20Preferences%2C%20by%20adding%20the%20app%20to%20Full%20Disk%20Access.).

I've completed the first of these steps now and everything is working. It would be nice if the application was able to prompt more explicitly about which of these two steps still needed to be completed, but also the steps were already listed in the README.

Thanks for the very quick response on this!
